snippets/microshift-rhde-compatibility-table-snip.adoc

```diff
@@ -7,29 +7,29 @@
 
 .{op-system-bundle} release compatibility matrix
 
-{op-system-base-full} and {microshift-short} work together as a single solution for device-edge computing. Supported configurations of {op-system-bundle} use verified releases for each together as listed in the following table:
+{op-system-base-full} and {microshift-short} work together as a single solution for device-edge computing. You can update each component separately, but the product versions must be compatible. For example, an update of {microshift-short} from 4.14 to 4.16 requires a {op-system} update. Supported configurations of {op-system-bundle} use verified releases for each together as listed in the following table:
```

Comment:
You can probably elaborate on compatibility between which versions we are referring to. Something similar to (if I am understanding correctly):
```suggestion
{op-system-base-full} and {microshift-short} work together as a single solution for device-edge computing. You can update each component separately, but the source version and target version must be compatible to each other. For example, an update of {microshift-short} from 4.14 to 4.16 requires a {op-system} update. Supported configurations of {op-system-bundle} use verified releases for each together as listed in the following table:
```

Reply:
@snarayan-redhat I can make the changes to lines 32, 27, 22 but for lines 10 and 17 I think can be retained as is. This is a mirror of the PR mentioned in additional information which is approved.
@ShaunaDiaz please suggest.

Reply:
source and target don't make sense here; let's leave this PM-approved text as-is